app/javascript/components/Calendar/index.js

```diff
@@ -158,4 +159,4 @@ const Calendar = ({
     </Layout>
   )
 
-export default Calendar
+export default MyCalendar
```

Comment:
Could we `import { Calendar as BigCalendar } from 'react-big-calendar'` to avoid renaming the component? Otherwise we introduce inconsistency between component name and file name.

Reply:
Sure, good idea (actually I did this as a temporary workaround, and then forget about it 🙁 )